Dockerfile

```diff
@@ -1,5 +1,5 @@
 # create a file named Dockerfile
-FROM node:argon
+FROM node:alpine
```

Comment:
https://hub.docker.com/_/node/

Had to update the version of node. VM2 needs 6.1 +